### ISSUE TITLE
FireFox: TypeError: activeMaskset.tests[testPos] is undefined

### ISSUE THREAD

**Issue by andywolk** [RobinHerbots/Inputmask#367]
Version: 2.3.68 jquery.inputmask.bundle.js line 542
This is all I have. Will provide more information if possible.

**Comment by RobinHerbots** [comment]
@andywolk ,

Can you give an example of the mask you are using.

**Comment by andywolk** [comment]
Okay. Probably solved. Looks like jquery 1.9.0 is a minimum version for me. Otherwise I have got the error message in a big project with the mask: (999)999-9999
but no error in a small reproducing script (sounds strange).

**Comment by RobinHerbots** [comment]
@andywolk , 

Can you give some example of other plugins  you use in the big project and in which sequence you are loading them.

**Comment by andywolk** [comment]
jquery
date.js http://www.JavascriptToolbox.com/lib/date/
jquery-ui.js 
jquery.clickmenu.js http://p.sohei.org
jquery.quickTree.js http://scottdarby.com/
jquery.validate.js http://jqueryvalidation.org/
jquery.inputmask.bundle.min.js

**Comment by andywolk** [comment]
Looks like something is wrong with a val() setter hook.
$(obj).val(value); // gives the error
$(obj).attr('value', value); // does not. But we should use val() with the latest jquery, not attr()

**Comment by RobinHerbots** [comment]
Probably a conflict in some valhook with the different plugins.

Just need to verify them one by one

2013/10/29 andywolk notifications@github.com

**Comment by andywolk** [comment]
Inputmask plugin uses hooks only.

**Comment by RobinHerbots** [comment]
We need some test page which has the problem, so I can investigate.

2013/10/29 andywolk notifications@github.com

**Comment by andywolk** [comment]
Have reproduced the issue. Somehow I missed the fact that my mask was empty in a few cases. So this is obviously my fault. I am sorry. Now I can give an example:
<html>
<head>
        <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js" type="text/javascript"></script>
        <script src="jquery.inputmask.bundle.js" type="text/javascript"></script>
</head>
<body>
    <br><input type="text" id="test" value="">

```
<script type="text/javascript">
    $(document).ready(function(){

        $("#test").inputmask(""); // Empty mask
        $("#test").val('123');

    });        
</script>
```

</body>
</html>

**Comment by RobinHerbots** [comment]
Great!

I will fix the case off an empty mask.

2013/10/29 andywolk notifications@github.com

**Comment by RobinHerbots** [comment]
Fixed in version 2.4.1
